row.py

```diff
@@ -795,6 +796,143 @@ def download_run(bucket, run_name):
     return location.joinpath(run_name)
 
 
+def download_ocr_results(bucket_name, run_name, out_dir):
+    """download ocr results from a GCP bucket
+
+    Args:
+        bucket (str): the name of the bucket
+        run_name (str): the name of the run to get files from
+        out_dir (str): where to save the results
+
+    Returns:
+        str: the location of the files
+    """
+    if bucket_name.startswith("gs://"):
+        bucket_name = bucket_name[5:]
+
+    if "STORAGE_CLIENT" not in globals():
+        STORAGE_CLIENT = google.cloud.storage.Client()
+    bucket = STORAGE_CLIENT.bucket(bucket_name)
+    blobs = bucket.list_blobs(prefix=run_name)
+    location = Path(__file__).parent / "data"
+
+    if not location.joinpath(f"ocr_results/{run_name}").exists():
+        location.joinpath(f"ocr_results/{run_name}").mkdir(parents=True)
+
+    ocr_dir = location.joinpath(f"ocr_results/{run_name}")
+
+    #: download .gz files
+    logging.info("downloading .gz files from cloud storage")
+    for blob in blobs:
+        if blob.name.endswith(".gz"):
+            print(blob.name)
+            blob.download_to_filename(ocr_dir / blob.name)
```

Comment:
**polish**: the list blobs iterator should only contain `.gz` files. If that is not the case then the filter should be updated. 

This could also be a comprehension. Ideally we update this so the if filter isn't necessary.

```suggestion
    [blob.download_to_filename(ocr_dir / blob.name) for blob in blobs if blob.name.endswith(".gz")]
```

Reply:
done